docs/DeveloperGuide.adoc

```diff
@@ -840,18 +1097,71 @@ Use case ends.
 * 3a. The given index is invalid.
 +
 [none]
-** 3a1. AddressBook shows an error message.
+** 3a1. CRM Book shows an error message.
 +
 Use case resumes at step 2.
 
-_{More to be added}_
+=== Use case: Detect a duplicate Lead or Contact
+
+*MSS*
+
+1. User enters a Lead to add
+2. CRM Book determines if Lead can be added
+3. CRM Book displays message on whether Lead entered can be added or is a duplicate
```

Comment:
Wrong sequence -- MSS should be Actor-System-Actor-System

Reply:
@Sheikh-Umar please resolve this

Reply:
I have resolved it.